### DISCUSSION TITLE
アプリケーション管理ページでアプリケーションを作成できない

### DISCUSSION THREAD

**Discussion by korinVR** [pixiv/vroid-sdk-developers#15] [Q&A] [answered]
お世話になります。https://hub.vroid.com/oauth/applications からアプリケーションを作成しようとしたのですが、下記の要領で「アプリケーション名」に何を入力してもエラーが出てしまいます。何か留意すべき注意事項などございますでしょうか。
ブラウザはWindows 10でChrome 94を使用しています。

![image](https://user-images.githubusercontent.com/882466/137239337-913e5489-1107-4103-9470-5f77c099d49d.png)

**Comment by Haresaku** [comment, marked as answer]
`VRoid` または `pixiv` が入っているアプリケーション名はアプリケーションとして登録することはできません。
お手数ですがそのワードの入っていないアプリケーション名で作成をお願いいたします。

**Comment by korinVR** [comment]
あっ、そういうことですね！　SDK名＋Test的な名前をつけることが多く、いくつか試した名前にすべて「VRoid」が入っておりました……。
無事作成できました。ありがとうございます！
